src/components/AnimatedLink.tsx

```diff
@@ -3,17 +3,27 @@ import NextLink from "next/link";
 import UpRightArrow from "public/assets/up-right-arrow.svg";
 import { ReactNode } from "react";
 
-export default function AnimatedLink({ href, children }: { href: string; children: ReactNode }) {
+export default function AnimatedLink({
+  href,
+  children,
+  asATag = false,
+}: {
+  href: string;
+  children: ReactNode;
+  asATag?: boolean;
+}) {
+  const Link = asATag ? "a" : NextLink;
```

Comment:
allow using a regular <a> tag to fix a weird scroll bug that seems to be caused by the next/link component itself